src/app/app.component.ts

```diff
@@ -37,4 +84,70 @@ export class AppComponent {
       this.userName = null;
     }
   }
+
+  loadCmsSeries(auth: HalDoc) {
+    auth.followItems('prx:series', {filters: 'v4'}).subscribe((series: HalDoc[]) => {
+      series.map(doc => {
+        return {
+          doc,
+          seriesId: doc['id'],
+          title: doc['title']
+        };
+      }).forEach(this.getSeriesPodcastDistribution.bind(this));
+    });
+  }
+
+  getSeriesPodcastDistribution(podcast: PodcastModel) {
```

Comment:
I think what we settled on is that Feeder does not have an endpoint that will give you the list of podcasts that a user is authorized for, so that's why we're needing to use CMS. The suggestion was to use a different user, but I'm not sure what other users are available to us that actually have podcasts with metrics data. TBH, I didn't really find any of that actionable.

Reply:
Yeah, that matches my recollection